src/klein/_interfaces.py

```diff
@@ -21,7 +21,7 @@
 from ._typing import ifmethod
 
 
-class IKleinRequest(Interface):
+class _IKleinRequest(Interface):
```

Comment:
Treat `IKleinRequest` the way we do other interfaces to appease Mypy.
This should go hopefully away when we adopt the Zope plugin.